### ISSUE TITLE
New project for accessibility findings

### ISSUE THREAD

**Issue by Klap-in** [dokuwiki/dokuwiki#4339]
### The problem

I cannot find the projects in this GitHub repository. I guess it has to do with the change to a new kind of project by GitHub? On of them was the WCAG audit of @selfthinker .
Does someone know how to make a new project, or would a milestone more fitting…

https://www.dokuwiki.org/wcag-audit

### Version of DokuWiki

Kaos

### PHP Version

8

**Comment by eightysixed** [comment]
Are you looking for [this?](https://github.com/dokuwiki/dokuwiki/issues/3464l)

**Comment by saschaleib** [comment]
There is also the [wcag-audit](https://www.dokuwiki.org/wcag-audit) page on the DokuWiki, er, wiki :-)

I am planning to take care of a couple of accessibility issues in the coming weeks. My focus will be on the ones that have to be fixed in the core, not so much the template, but I will of course look at these, too.

That will however raise another issue, and that is that these changes will change the HTML code that is generated by the DokuWiki core, and that may break (or at least cause issues with) some of the existing templates.

It is really up to @splitbrain to decide how much breaking things is acceptable here. I am aware that this is a difficult balance to strike, and I will of course understand if not all of those changes will make it into the next version :-)

**Comment by splitbrain** [comment]
> I cannot find the (old) projects in this GitHub repository

Hmm.. it seems projects were disabled in the organization. I enabled them but the projects we had are still nowhere to be seen. Either the were lost on the migration to the organization or they were deleted when Github introduced their new projects feature.

All accessibility tickets @selfthinker created are easily found via the accessibility tag. Should be simple to readd them to a new project if wanted.
